### DISCUSSION TITLE
Find tool spacing issue

### DISCUSSION THREAD

**Discussion by othon-lima** [stephanrauh/ngx-extended-pdf-viewer#1520] [Q&A] [answered]
The find tool only finds words in separate lines if I enter it without space between the them, as shown in the pictures.

Is there any way I can find the words without have to enter them with no space in between?

Thanks!

![SearchFound](https://user-images.githubusercontent.com/73144452/188215945-91b925ae-a576-4ea2-a42b-9a4e59266e7f.png)
![SearchNotFound](https://user-images.githubusercontent.com/73144452/188215946-3b7a3f45-0945-4cee-8056-2f680bcf8926.png)

**Comment by stephanrauh** [comment, marked as answer]
Sounds like a duplicate of #1449. I've solved that bug with version 14.5.3. Which version are you using?

**Reply by othon-lima** [reply]
Sorry I didn't find that issue when looking for this problem. I was using version 13.*. 

I just updated and it worked as intended. Thank you so much for your help!

**Reply by stephanrauh** [reply]
Hooray!
